docs/src/languages/go.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Go"
-weight: 2
+weight: 3
```

Comment:
Wouldn't it be better to specify no weight on these and let them all sort alphabetically on their own?

Reply:
Absolutely. On my list. I'll change it to default alphabetical unless weights are present.

Reply:
Oh, look at  that. Was so focused on getting weights  to work where we don't want them alphabetical, didn't notice it will already be  alphabetical by default. Updated for languages.